bin/run_busco.sh

```diff
@@ -0,0 +1,155 @@
+            # TODO double-check if selected dataset is not one of bacteria_*, archaea_*, eukaryota_*?
+            echo "Domain could not be selected by BUSCO, but virus dataset was selected."
+            cp BUSCO/short_summary.specific.${db_name_spec}.BUSCO.txt short_summary.specific_lineage.${db_name_spec}.${bin}.txt
+        else
+            echo "ERROR: Some not expected case occurred! See ${bin}_busco.log." >&2
+            exit 1
+        fi
+    fi
+
+    for f in BUSCO/run_${db_name_spec}/busco_sequences/single_copy_busco_sequences/*faa; do
+        cat BUSCO/run_${db_name_spec}/busco_sequences/single_copy_busco_sequences/*faa | gzip >${bin}_buscos.${db_name_spec}.faa.gz
+        break
+    done
+    for f in BUSCO/run_${db_name_spec}/busco_sequences/single_copy_busco_sequences/*fna; do
+        cat BUSCO/run_${db_name_spec}/busco_sequences/single_copy_busco_sequences/*fna | gzip >${bin}_buscos.${db_name_spec}.fna.gz
+        break
+    done
+
+elif egrep -q $'ERROR:\tNo genes were recognized by BUSCO' ${bin}_busco.err ; then
+    echo "WARNING: BUSCO analysis failed due to no recognized genes! See also ${bin}_busco.err."
+    echo -e "${bin}\tNo genes" > "${bin}_busco.failed_bin.txt"
+
+elif egrep -q $'INFO:\t\\S+ selected' ${bin}_busco.log && egrep -q $'ERROR:\tPlacements failed' ${bin}_busco.err ; then
+    echo "WARNING: BUSCO analysis failed due to failed placements! See also ${bin}_busco.err. Still using results for selected generic lineage dataset."
+    echo -e "${bin}\tPlacements failed" > "${bin}_busco.failed_bin.txt"
+
+    message=$(egrep $'INFO:\t\\S+ selected' ${bin}_busco.log)
+    [[ $message =~ INFO:[[:space:]]([_[:alnum:]]+)[[:space:]]selected ]];
+    db_name_gen="${BASH_REMATCH[1]}"
+    most_spec_db=${db_name_gen}
+    echo "Used generic lineage dataset: ${db_name_gen}"
+    cp BUSCO/auto_lineage/run_${db_name_gen}/short_summary.txt short_summary.domain.${db_name_gen}.${bin}.txt
+
+    for f in BUSCO/auto_lineage/run_${db_name_gen}/busco_sequences/single_copy_busco_sequences/*faa; do
+        cat BUSCO/auto_lineage/run_${db_name_gen}/busco_sequences/single_copy_busco_sequences/*faa | gzip >${bin}_buscos.${db_name_gen}.faa.gz
+        break
+    done
+    for f in BUSCO/auto_lineage/run_${db_name_gen}/busco_sequences/single_copy_busco_sequences/*fna; do
+        cat BUSCO/auto_lineage/run_${db_name_gen}/busco_sequences/single_copy_busco_sequences/*fna | gzip >${bin}_buscos.${db_name_gen}.fna.gz
+        break
+    done
+
+else
+    echo "ERROR: BUSCO analysis failed for some unknown reason! See also ${bin}_busco.err." >&2
+    exit 1
+fi
+
+# additionally output genes predicted with Prodigal (GFF3)
+if [ -f BUSCO/logs/prodigal_out.log ]; then
+    mv BUSCO/logs/prodigal_out.log "${bin}_prodigal.gff"
+fi
+
+# if needed delete temporary BUSCO files
+if [ ${busco_clean} = "Y" ]; then
+    find . -depth -type d -name "augustus_config" -execdir rm -rf "{}" \\;
```

Comment:
The semi-colon here is double `\\` escaped in these lines, as `\;` causes parsing errors when in a Nextflow script block. In a plain bash script I think this should just be single-escaped - might be what's causing the execdir error.

Otherwise, changes look good to me 👍

Reply:
GAH! I had to miss one didn't I...